core/api/api.hpp

```diff
@@ -246,6 +247,11 @@ namespace fc::api {
     uint64_t size;
   };
 
+  struct AddChannelInfo {
```

Comment:
Rename to `ChannelInfo`

Reply:
ChannelInfo is misleading, there is one `core/payment_channel_manager/payment_channel_manager.hpp`.